Editor/Deployment/DeploymentSettings.cs

```diff
@@ -174,7 +174,10 @@ public async Task SetGameNameAsync(string value)
 
             _gameName = value;
             ClearCurrentStackInfo();
-            await _delayedStackInfoRefresh.Request();
+            if (_stateManager.IsBootstrapped)
```

Comment:
I'm not sure i understand this condition. Even if the profile is bootstrapped is it not possible that the stack information doesn't exist until the "Create" button is clicked in the Managed EC flow?

Reply:
The code that generates the warning is the following

https://github.com/aws/amazon-gamelift-plugin-unity/blob/9ef50c7b98e4497c4a59d9632a98b58ad54ab700/Editor/Deployment/DeploymentSettings.cs#L181-L218

If we are bootstrapped it would not encounter the initial checks

Reply:
Why have we added the check to this method? It seems fairly arbitrary. Why is SetGameName even being called before the profile is bootstrapped?

This log indicates we're hitting a call path that wasn't intended, so this check is more of a bandaid solution right now.